css/styles.css

```diff
@@ -113,3 +121,90 @@ header, footer {
 .tt-cursor, .tt-suggestion:hover {
     background-color: #efefef;
 }
+
+/* improvements by YuriBrunetto */
+hgroup {
+    margin-top: 30px;
+}
+h2 {
+    font-weight: normal;
+    font-size: 16px;
+    color: #666;
```

Comment:
Please update all occurrences of `#666` :see_no_evil:, also you can just use multiple classes on the elements that need this color so it's not re-declared so many times